config/puma.rb

```diff
@@ -44,6 +44,6 @@
 plugin :tmp_restart
 
 # Metrics
-activate_control_app
+activate_control_app('unix://./tmp/sockets/puma.sock')
```

Comment:
Would this work for forked workers?

Reply:
Yes, puma has this solved for forked workers, you only control the parent process and not the children.

Reply:
`./` bounds it to CWD. Shouldn't it be specific apps relative location?

Reply:
CWD here is the specific app's relative location, Rails.root is not available in this context. Do you have any other suggestions how to put it into the `./tmp` folder?

Reply:
Maybe from the file location?

Reply:
Seems like an overkill, but okay